code/modules/client/preference_setup/loadout/loadout_eyes.dm

```diff
@@ -134,16 +134,30 @@
 	gear_tweaks += new /datum/gear_tweak/path(blindfold)
 
 /datum/gear/eyes/goon_goggles
-	display_name = "tactical goggles selection"
+	display_name = "tactical HUD goggles selection (security)"
 	description = "A selection of tactical eyewear. Note that factional ones can only be taken by members of that faction."
 	path = /obj/item/clothing/glasses/safety/goggles/goon
 
 /datum/gear/eyes/goon_goggles/New()
-	allowed_roles = security_positions
 	..()
+	allowed_roles = security_positions
```

Comment:
consider moving this up under `/datum/gear/eyes/goon_goggles`.

Reply:
It doesn't work if moved above and causes runtime errors.